includes/definitions/discovery/panos.yaml

```diff
@@ -32,9 +32,15 @@ modules:
         count:
             data:
                 -
-                    oid: panSession
-                    value: panSessionActive
-                    num_oid: '.1.3.6.1.4.1.25461.2.1.2.3.{{ $index }}'
+                    oid: panSessionActive
```

Comment:
The old code walked panSession so it would include both panSessionActive and panSessionMax.  How are your changes an improvement?

Reply:
Because the old code doesn't work, always at 0, look at the LibreNMS community post, i't a duplicate of the usage of panSessionActive